pennylane_lightning/core/_serialize.py

```diff
@@ -47,14 +48,22 @@
 }
 
 
+def _chunk_iterable(iteration, num_chunks):
```

Comment:
Are we removing this from its other defined location? Could we import it instead?